middlewares/campaigns.js

```diff
@@ -8,6 +8,8 @@ const errors = require('../lib/errors');
 const helpers = require('../lib/helpers');
 const constants = require('../lib/constants');
 const { sequelize } = require('../lib/sequelize');
+const superagent = require('superagent');
```

Comment:
1) we already have request and request-promise-native, can we use this?
2) eslint rule: global deps first, then local ones
3) better to move it to other file, which will be the wrapper for gsuite-wrapper requests and use it there, check mailer.js for reference

Reply:
1) yes definitely, i was improvising
2) yes, I will lint too (it is still incomplete anyway)
3) move what?

Reply:
on 3), we have the payload generation and actual query inside the middleware, which I think doesn't belong here. instead, we can create a module (like `lib/gsuite.js`) which will contain functions that are wrappers for gsuite-wrapper requests.
check out how I call functions from mailer.js for the idea

Reply:
1. request (and therefore request-promise-native) are apparently deprecated. So we'll have to look into an alternative

Reply:
I'd say moving to another fetching library can be done in another task, my main point here is that in this PR we have 2 libs that do pretty much the same instead of one, and request-promise-native is already used everywhere in the project.
We should plan moving to something else (axios?) at some point.

Reply:
Agree, changing the fetching library is something for in the backlog